ext/pcaprub_c/pcaprub.c

```diff
@@ -407,7 +410,7 @@ rbpcap_setfilter(VALUE self, VALUE filter)
 {
   char eb[PCAP_ERRBUF_SIZE];
   rbpcap_t *rbp;
-  u_int32_t mask = 0, netid = 0;
+  uint32_t mask = 0, netid = 0;
```

Comment:
Just to confirm; was `u_int32_t` used for older compiler environments that don't have `uint32_t`? From a quick Google that might be from the windows 7 era?

If so - do we need to be more clever here to support backwards compatibility 👀

Reply:
I think it's actually more correct to use `bpf_u_int32` so I've made those changes
https://www.tcpdump.org/manpages/pcap_compile.3pcap.html
https://www.tcpdump.org/manpages/pcap_lookupnet.3pcap.html

Reply:
LGTM